client/gatsby-ssr.js

```diff
@@ -32,3 +32,19 @@ export const onRenderBody = ({
   setHeadComponents(getheadTagComponents());
   setPostBodyComponents(getPostBodyComponents(pathname));
 };
+
+export const onPreRenderHTML = ({
+  getHeadComponents,
+  replaceHeadComponents
+}) => {
+  const headComponents = getHeadComponents();
+  headComponents.sort((x, y) => {
+    if (x.key === 'boostrap-min-preload' || x.key === 'boostrap-min') {
+      return -1;
+    } else if (y.key === 'TypographyStyle' || x.key === 'boostrap-min') {
```

Comment:
```suggestion
    } else if (y.key === 'bootstrap-min-preload' || y.key === 'boostrap-min') {
```
I think the logic here is 
> if (x is bootstrap) then x comes first
> if (y is bootstrap) then y comes first

and we don't care about the order other than the two bootstraps should be first.

Reply:
this should be resolved.